src/main/resources/de/unijena/cheminf/mortar/message/Message_en_GB.properties

```diff
@@ -544,4 +544,24 @@ IMoleculeFragmenter.CycleFinderOption.relevant.tooltip = Relevant cycles of a gr
 IMoleculeFragmenter.CycleFinderOption.tripletShort.displayName = triplet short
 IMoleculeFragmenter.CycleFinderOption.tripletShort.tooltip =  The shortest cycle through each triple of vertices; allows to generate the envelope rings of some molecules (e.g. naphthalene) without generating all cycles
 IMoleculeFragmenter.CycleFinderOption.vertexShort.displayName = vertex short
-IMoleculeFragmenter.CycleFinderOption.vertexShort.tooltip = The shortest cycles through each vertex; linear independence is not checked; in practice similar to MCB
\ No newline at end of file
+IMoleculeFragmenter.CycleFinderOption.vertexShort.tooltip = The shortest cycles through each vertex; linear independence is not checked; in practice similar to MCB
+##AlkylStructureFragmenter##
+AlkylStructureFragmenter.displayName = Alkyl Fragmenter
+AlkylStructureFragmenter.fragmentSaturationSetting.tooltip = Defines how open valences resulting from bond breakages during fragmentation should be saturated
+AlkylStructureFragmenter.fragmentSaturationSetting.displayName = Fragment saturation setting
+AlkylStructureFragmenter.keepNonFragmentableMoleculesSetting.tooltip = If set to true, enables non-fragmentable molecules to stay in the fragmenter pipeline and be returned together with all received fragments.
+AlkylStructureFragmenter.keepNonFragmentableMoleculesSetting.displayName = Keep non-fragmentable molecules in Pipeline
+AlkylStructureFragmenter.maxChainLengthSetting.tooltip = Defines maximum length of chains: 0 is no restrictions, 1 returns methanes
+AlkylStructureFragmenter.maxChainLengthSetting.displayName = Limit length of returned chains
+AlkylStructureFragmenter.fragmentSideChainsSetting.tooltip = Defines whether chains should be fragmented
+AlkylStructureFragmenter.fragmentSideChainsSetting.displayName = Fragment linear chains
+AlkylStructureFragmenter.altHandlingSingleTertQuatCarbonsSetting.tooltip = Isolates tertiary and quaternary carbons from non-cyclic chains and saturates them with CDK pseudo atoms representing the former neighbor atoms accordingly (Example represented in SMILES notation: CC(C)(C)C.*C(*)(*)* ).
+AlkylStructureFragmenter.altHandlingSingleTertQuatCarbonsSetting.displayName = Isolate tertiary and quaternary carbons
```

Comment:
... setting

Reply:
this sounds like tert. and quat. carbons are extracted or not but it is about the environment their are extracted with!